app/code/Magento/InventoryInStorePickup/Model/PickupLocation/GetPickupLocationByCode.php

```diff
@@ -0,0 +1,56 @@
+<?php
```

Comment:
I think we does not need one more service for these purposes.
We already have a service which can provide Pickup Location based on the code and Sales Channel.

Reply:
ok